missions/views/question.py

```diff
@@ -31,7 +39,11 @@ class DeleteView(LoginRequiredMixin, PermissionRequiredMixin, generic.DeleteView
 
 class UpdateView(LoginRequiredMixin, PermissionRequiredMixin, generic.UpdateView):
     model = Question
-    fields = ['text', 'mandatory']
+    fields = get_all_languages_fieldname('text')
```

Comment:
You can do something like `fields = [*get_all_languages_fieldname('text'), 'mandatory']`. Avoids the ugly side-effect.

Reply:
done.